server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -679,6 +688,7 @@ public void facet_on_created_at_with_less_than_20_weeks() {
       entry("2014-09-15T01:00:00+0000", 1L));
   }
 
+  @Ignore
   @Test
   public void facet_on_created_at_with_less_than_20_months() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)